.buildkite/scripts/build_component.sh

```diff
@@ -96,7 +95,7 @@ esac
 
 # TODO (CM): Replace "Linux" below with ${pkg_target:?} once that's in
 # hab-plan-build (see https://github.com/habitat-sh/habitat/pull/5373)
-echo -e "<br>* ${pkg_ident} (Linux)" | buildkite-agent annotate --append --context "release-manifest"
+echo "<br>* ${pkg_ident} (Linux)" | buildkite-agent annotate --append --context "release-manifest"
```

Comment:
I wonder if using both an html `<br>` (maybe `<br/>`?) and the Markdown `*` is causing the Buildkite interpreter to get confused? Although as I recall, all HTML tags are valid in Markdown, so who knows